internal/hud/view/view.go

```diff
@@ -66,6 +66,17 @@ func (YAMLResourceInfo) resourceInfoView()              {}
 func (yamlInfo YAMLResourceInfo) RuntimeLog() model.Log { return model.NewLog("") }
 func (yamlInfo YAMLResourceInfo) Status() string        { return "" }
 
+type LocalResourceInfo struct {
+}
+
+var _ ResourceInfoView = LocalResourceInfo{}
+
+var LocalResourceStatusPlaceholder = "local-resource-ok"
+
+func (LocalResourceInfo) resourceInfoView()     {}
+func (LocalResourceInfo) RuntimeLog() model.Log { return model.Log{} }                    // no runtime log, only build log
```

Comment:
This is a bit weird, but I'm not sure that it matters, in that really a local resource _only_ has a runtime log, and no build log. But if it's just naming and doesn't semantically matter then it's neither here nor there.

Reply:
Aaah, i beg to differ! Because of weird semantics, the "build" step is what happens when the resource goes thru our BuildAndDeployer--i.e. we run the command during the BUILD step. That's where any logs/errors will come out. Runtime log/runtime status indicate the life of the resource _beyond_ the "build" step, and in the case of LocalResource currently, there is none.

Reply:
Right, OK!